### PR TITLE
[test][sanity] whitelist openssl cve vulnerability installed using apt

### PR DESCRIPTION
**Pull request by tejaschumbalkar**
*GitHub Issue #, if available:*

Note: If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

### Description
- whitelist openssl [CVE-2021-3711](https://nvd.nist.gov/vuln/detail/CVE-2021-3711) vulnerability installed on DLC using apt 

## Label Checklist
- [ ] I have added the project label for this PR (*<project_name>* or "Improvement")

## PR Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [ ] If the PR changes affects SM test, I've modified dlc_developer_config.toml in my PR branch by setting sagemaker_tests = true and efa_tests = true
- [ ] If this PR changes existing code, the change fully backward compatible with pre-existing code. (Non backward-compatible changes need special approval.)
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Pytest Marker Checklist
- [ ] (If applicable) I have added the marker `@pytest.mark.model("<model-type>")` to the new tests which I have added, to specify the Deep Learning model that is used in the test (use `"N/A"` if the test doesn't use a model)
- [ ] (If applicable) I have added the marker `@pytest.mark.integration("<feature-being-tested>")` to the new tests which I have added, to specify the feature that will be tested
- [ ] (If applicable) I have added the marker `@pytest.mark.multinode(<integer-num-nodes>)` to the new tests which I have added, to specify the number of nodes used on a multi-node test
- [ ] (If applicable) I have added the marker `@pytest.mark.processor(<"cpu"/"gpu"/"eia"/"neuron">)` to the new tests which I have added, if a test is specifically applicable to only one processor type

#### EIA/NEURON Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `ei_mode = true` or `neuron_mode = true`

#### Benchmark Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `benchmark_mode = true`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/dlc_tests/sanity/test_pre_release.py [hunk 1]
```diff
@@ -238,6 +239,26 @@ def _run_dependency_check_test(image, ec2_connection, processor):
         "CVE-2020-13936",
     }
 
+    # Whitelist CVE #CVE-2021-3711 for DLCs where openssl is installed using apt-get
+    framework, fw_version = get_framework_and_version_from_tag(image)
+
+    references = { 
+        "tensorflow2": ["2.3.4", "2.4.3", "2.6.0"],
```
Comment (jeet4320) [inline test/dlc_tests/sanity/test_pre_release.py:246]: can we do just `2.3, 2.4` instead of mentioning patch version else it would need changes for every patch version update, we use same docker file for patch version update?

Comment (tejaschumbalkar) [reply test/dlc_tests/sanity/test_pre_release.py]: yes, we do, pushed  the change

**Review by jeet4320 — DISMISSED** [review]

**Review by jeet4320 — APPROVED** [review]

File: test/dlc_tests/sanity/test_pre_release.py [hunk 2]
```diff
@@ -238,6 +239,29 @@ def _run_dependency_check_test(image, ec2_connection, processor):
...
 
+    # Whitelist CVE #CVE-2021-3711 for DLCs where openssl is installed using apt-get
+    framework, _ = get_framework_and_version_from_tag(image)
+    short_fw_version = re.search(r"(\d+\.\d+)", image).group(1)
+
+    references = { 
+        "tensorflow2": ["2.3", "2.4", "2.6"], 
+        "tensorflow1": ["1.15"], 
+        "mxnet": ["1.9"], 
+        "pytorch": [],
```
Comment (tejaschumbalkar) [inline test/dlc_tests/sanity/test_pre_release.py:250]: Removing PT versions as it is not affected in the first place.

File: test/dlc_tests/sanity/test_pre_release.py [hunk 3]
```diff
@@ -238,6 +239,29 @@ def _run_dependency_check_test(image, ec2_connection, processor):
...
+        "huggingface_pytorch": [],
+        "huggingface_tensorflow": []
```
Comment (tejaschumbalkar) [inline test/dlc_tests/sanity/test_pre_release.py:252]: Huggingface are affected since it consumes the upstream DLC images

**Review by jeet4320 — APPROVED** [review]
Ok to merge
